### ISSUE TITLE
Accessibility UI: Options of drop down boxes in dark mode are not visible accessible due to wrong cloror settings

### ISSUE THREAD

**Issue by tigexplorer** [Chocobozzz/PeerTube#6031]
### Describe the current behavior

The options texts in drop down boxes are no readable. They are styled with wrong colors (bright text on bright background)

### Steps to reproduce

1. Log in as a user or admin
2. Click for options on any drop down box
3. The options are displayed with bright text color of bright background color
4. The options texts are not readable

### Describe the expected behavior

The drop down options may styled with bright text on dark background color

### Additional information

* PeerTube instance: 
  * URL: https://tube.pilgerweg-21.de
  * Version: 5.2.1
  * NodeJS version:
  * Ffmpeg version:

* Browser name, version and platforms on which you could reproduce the bug: Firefox 119.0.1

![Bildschirmfoto vom 2023-11-15 11-15-51](https://github.com/Chocobozzz/PeerTube/assets/43775552/b781c7cb-9491-4002-8c62-fdcdbc2d6bce)

* Link to browser console log if relevant:
* Link to server log if relevant (`journalctl` or `/var/www/peertube/storage/logs/`):

**Comment by Booteille** [comment]
Hi!

I can't reproduce this on nightly and with a fresh dark theme installed.
Maybe it had been fixed since.

Could you try to be sure you installed the latest dark-theme, first?
Also, if you can try with the v6.0.0-rc1 to check if your issue has been resolved since, that would be nice.
If you can't, just wait some weeks the stable version releases and upgrade.

Don't forget to read release notes to avoid any issue during the upgrade!

**Comment by tigexplorer** [comment]
> Hi!
> 
> I can't reproduce this on nightly with a fresh dark theme installed. Maybe it had been fixed since the version you're using.
> 
> Could you try to be sure you installed the latest dark-theme, first? Also, if you can try with the v6.0.0-rc1 to check if your issue has been resolved since, that would be nice. If you can't, just wait some weeks the stable version releases and upgrade.
> 
> Don't forget to read release notes to avoid any issue during the upgrade!

Thanx for your quick reply and your work.

I'm using the last dark theme version but being unable to upgrade peertube main to nightly. I will wait for the next release.
